app/services/import/persist/persist-licence-versions.service.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+/**
+ * Creates or updates an imported licence's versions and its child entities that have been transformed and validated
+ * @module PersistLicenceVersionsService
+ */
+
+const LicenceVersionPurposeConditionModel = require('../../../models/licence-version-purpose-condition.model.js')
+const LicenceVersionPurposeModel = require('../../../models/licence-version-purpose.model.js')
+const LicenceVersionModel = require('../../../models/licence-version.model.js')
+
+/**
+ * Creates or updates an imported licence's versions and its child entities that have been transformed and validated.
+ *
+ * @param {object} trx - An Objection.js transaction object for PostgreSQL.
+ * @param {string} updatedAt - The timestamp indicating when the entity was last updated.
+ * @param {object} transformedLicence - An object representing a valid WRLS licence.
+ * @param {string} licenceId - A licence id from WRLS
+ *
+ */
+async function go (trx, updatedAt, transformedLicence, licenceId) {
+  await _persistLicenceVersions(trx, updatedAt, transformedLicence.licenceVersions, licenceId)
```

Comment:
We can just return this.

```suggestion
  return _persistLicenceVersions(trx, updatedAt, transformedLicence.licenceVersions, licenceId)
```